docker-compose.yaml

```
@@ -0,0 +1,46 @@
+version: '3.2'
+
+services:
+  elasticsearch:
+    image: elasticsearch:7.17.6
+    ports:
+      - "9200:9200"
+      - "9300:9300"
+    environment:
+      ES_JAVA_OPTS: "-Xmx256m -Xms256m"
+      ELASTIC_PASSWORD: elastic
+      discovery.type: single-node
+    healthcheck:
+        test: ["CMD", "curl", "-f", "http://127.0.0.1:9200/_cluster/health?wait_for_status=green"]
+        # Otherwise elasticsearch cannot dl the geoip map because mozi take all the bandwidth
+        interval: 50s
+        timeout: 30s
+        retries: 5
+    networks:
+      - elk
+
+  kibana:
+    image: kibana:7.17.6
+    ports:
+      - "5601:5601"
+    environment:
+      - ELASTICSEARCH_USERNAME=elastic
+      - ELASTICSEARCH_PASSWORD=elastic
+    networks:
+      - elk
+    depends_on:
+      - elasticsearch
+
+  mozitools:
+    image: mozitools
```

```suggestion
    image: mozitools
    build:
      context: .
      dockerfile: Containerfile
```
This way, the mozitools image will be automatically built if it does not exists.